etc/profile-a-l/clion.profile

```diff
@@ -5,13 +5,13 @@ include clion.local
 # Persistent global definitions
 include globals.local
 
-nodeny  ${HOME}/.CLion*
```

Comment:
Reason? If this isn't used any more but in previous versions, there is no harm if we keep it and old versions might still be used.

Reply:
what kinda version was that?

Reply:
IDK, I'd never used CLion. But if this line is there, it has likely a reason and removing it could maybe break things for nearly no security gain.

Reply:
I just tried clion 1.1.2 and it's creating ~/.clion11 folder in home directory

Reply:
Then add the upper-case version back and add the lower-case version too.

Reply:
But it's working correctly without that folder

Reply:
Did you tested every single feature? What's in this folder?

Reply:
it's fetching correctly for example the theme without that folder
![image](https://user-images.githubusercontent.com/37552959/124753702-d8890680-df18-11eb-96d8-1d47040110ee.png)

Reply:
> **Did you tested _every_ single feature?**

-------

> [IMAGE]

OK, it saves the theme name there. And what else?

Reply:
All config variables there
Like installed and disabled plugins(working).

Reply:
[output.log](https://github.com/netblue30/firejail/files/6778487/output.log)